tsconfig.json

```diff
@@ -32,6 +32,10 @@
     "test-packages/*/node_modules",
     "tmp",
     "tests/scenarios/output",
+    "packages/config-meta-loader/src",
```

Comment:
does it still work without this line?

Reply:
No, if I don't ignore my package in the root `tsconfig.json`, then that config prevails and my package is compiled as CommonJS.

Reply:
sorry I misread this, I assumed this was the **include** block. This is totally perfect and I'm very happy with it 🎉